Samples/iOS-Swift/iOS-SwiftUITests/TopViewControllerTests.swift

```diff
@@ -0,0 +1,89 @@
+import XCTest
+
+class TopViewControllerTests: XCTestCase {
+    private let app: XCUIApplication = XCUIApplication()
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        XCUIDevice.shared.orientation = .portrait
+        app.launchEnvironment["io.sentry.ui-test.test-name"] = name
+        app.launch()
+        
+        waitForExistenceOfMainScreen()
+    }
+    
+    override func tearDown() {
+        app.terminate()
+        super.tearDown()
+    }
+    
+    func testTabBarViewController() {
+        openInspector()
+        
+        let getTopBT = app.buttons["BTN_TOPVC"]
```

Comment:
`l`: I think we don't need to shorten button. For me the following is easier to read. Just a personal preference. The same applies to lbl. I would call it `LABEL_TOPVC`
```suggestion
        let getTopBT = app.buttons["BUTTON_TOPVC"]
```

Reply:
Sorry, I will let this one slide.